src/components/icon/icon.style.js

```diff
@@ -55,24 +55,24 @@ const StyledIcon = styled.span`
 
     try {
       if (disabled) {
-        finalColor = theme.icon.disabled;
-        finalHoverColor = theme.icon.disabled;
+        finalColor = "var(--colorsYin030)";
+        finalHoverColor = "var(--colorsYin030)";
       } else if (color) {
         const { color: renderedColor } = styledColor({ color, theme });
```

Comment:
We're still using theme on this line and in multiple other places in this component, they should be changed to use tokens in this PR I believe